shared/js/sidebar.jsx

```diff
@@ -0,0 +1,179 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Now I'm getting into it, this file feels like a copy of conversation.jsx. I'm thinking it should therefore be a `git mv` rather than a copy.